src/CTA.WebForms/Helpers/TagConversion/TagConfigParser.cs

```diff
@@ -45,9 +45,14 @@ public TagConfigParser(string configsDir)
                     var tagName = GetTagNameForFile(filePath);
                     var converter = GetConverterForFile(filePath);
 
-                    // TODO: Validate converter here
-
-                    result.Add(tagName, converter);
+                    if (converter.Validate())
+                    {
+                        result.Add(tagName, converter);
+                    }
+                    else
+                    {
+                        LogHelper.LogError($"{Rules.Config.Constants.WebFormsErrorTag}Failed to validate generated converter for config at {filePath}");
+                    }
```

Comment:
Since failed validations are considered errors, we should also log the parameters that caused the error. We can do this by either logging the errors within the converters or have `Validate()` throw an exception with a detailed error message and catch/log the exception here.

Reply:
Good catch, made the change